applications/tari_dan_wallet_cli/src/command/transaction.rs

```diff
@@ -118,6 +122,16 @@ pub struct SubmitManifestArgs {
     common: CommonSubmitArgs,
 }
 
+#[derive(Debug, Args, Clone)]
+pub struct ClaimBurnArgs {
+    account_address: ComponentAddress,
+    commitment_address: String,
+    range_proof: String,
+    proof_of_knowledge: String,
```

Comment:
```suggestion
    commitment_address: FromHex<PublicKey>, // For this one, you may be able to uncomment the Hex impl in applications/tari_dan_wallet_cli/src/from_hex.rs
    range_proof: FromHex<Vec<u8>>,
    proof_of_knowledge: FromHex<Vec<u8>>,
```

Reply:
As suggested, used a `FromBase64` interface, similar to `FromHex` :)